app/models/incident_report.py

```diff
@@ -22,6 +22,8 @@ class IncidentReport(db.Model):
                                backref='incident_report')
     date = db.Column(db.DateTime)  # hour the incident occurred
     duration = db.Column(db.Interval)  # like timedelta object
+    # agency = TODO: Hook this up with agency when Nancy finishes. TODO: set
```

Comment:
I just tried that and it didn't fix it. I will cherry-pick these commits onto the `agency-schema` branch and re-push this.